src/main/java/com/drinchev/projectlabel/ProjectLabel.java

```diff
@@ -1,21 +1,33 @@
 package com.drinchev.projectlabel;
 
+
+import com.drinchev.projectlabel.resources.ui.ProjectLabelBackgroundImage;
 import com.drinchev.projectlabel.resources.ui.ProjectLabelStatusBarWidget;
 import com.intellij.openapi.components.Service;
+import com.intellij.openapi.components.Service.Level;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.wm.StatusBar;
 import com.intellij.openapi.wm.WindowManager;
+import org.jetbrains.annotations.NotNull;
+
+
+import static java.util.Objects.requireNonNull;
 
-@Service
-public final class ProjectLabelService {
+@Service(Level.PROJECT)
```

Comment:
Do you have any specific reason for making this a Project Service, rather than being an Application one?

Being an Application Level Service allows us to change the global setting of the plugin and it will affect all open projects.

Reply:
I've made it a Project Service, since it only acts on project resources. I've just tried to change it to `APP` or without any specification locally - but that does not result in affecting all open projects. Is that something that worked before (meaning I broke it)?